openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/pipelineDetails/PipelineDetailsPageTest.java

```diff
@@ -122,8 +128,15 @@ public void editTaskDescription() throws InterruptedException {
         .moveToElement(webDriver.findElement(By.xpath("//div[@data-testid='description']/button")))
         .perform();
     Events.click(webDriver, By.xpath("//div[@data-testid='description']/button"));
-    Events.sendKeys(webDriver, By.xpath(enterDescription), faker.address().toString());
+    Events.sendKeys(webDriver, By.xpath(enterDescription), editDescription);
     Events.click(webDriver, By.cssSelector("[data-testid='save']"));
+//    webDriver.navigate().refresh();
+//    Thread.sleep(1000);
+//    actions
+//        .moveToElement(webDriver.findElement(By.xpath("//div[@data-testid='description']/button")))
+//        .perform();
+//    Events.click(webDriver, By.xpath("//div[@data-testid='description']/button"));
+//    webDriver.findElement(By.xpath("//*[text()[contains(.,'" + editDescription + "')]] "));
```

Comment:
If the comments are not required, then lets remove it

Reply:
Addressed in the latest commit.